.github/workflows/build.yml

```diff
@@ -37,7 +37,7 @@ jobs:
           devScript: ./dev.sh
 
         - runtime: win-x64
-          os: windows-latest
+          os: windows-2019
```

Comment:
`windows-latest` is using `windows-2022` which doesn't not have .net 4.5 installed, change back to `windows-2019` for now